portal/messages/en.json

```
@@ -45,6 +45,7 @@
     "unsupported-chain-heading": "You're not connected to a supported chain",
     "wait-hours": "Wait ~<hours></hours>",
     "wait-minutes": "Wait ~{minutes} minutes",
```

In `portal/app/[locale]/tunnel/_components/evmWithdrawalWaitTime.tsx` this is called with

```tsx
{tCommon('wait-minutes', {
  minutes: 1,
})}
```

That means it will render `Wait ~1 minutes`, when it should be `Wait ~1 minute` (notice the singular version)

We need to implement the plural / singular form  (see example on line 49). I think it should be something like this

```suggestion
    "wait-minutes": "Wait ~{minutes, plural, =1 {# minute} other {# minutes}}",
```

This is how it should look like for Spanish

```js
"wait-minutes": "Espere hasta ~{minutes, plural, =1 {# minuto} other {# minutos}}",
```

I'll let you the do Portuguese version

See [next-intl docs for plural usage](https://next-intl.dev/docs/usage/messages#cardinal-pluralization)